concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -37,6 +37,29 @@ class CompletedProcessWithError:
     return CompletedProcessWithError()
 
 
+class Image:
```

Comment:
Why adding this class? Wouldn't be better to return `ndarray` in `MockedCV2`? Or better, prepare some mock, instead of addding stub classes?

Reply:
Could you provide an example ? My goal here is that there are several tests in which using `MockedCV2.imread` should return different results.

Reply:
you could have sth like this utility function/method:
```python
    def _prepare_cv2_mock(self, desired_behaviour='none'):
        mocked_cv2 = mock.Mock()
        mocked_cv2.imread = mock.create_autospec(cv2.imread, spec_set=True)
        if desired_behaviour == 'none':
            mocked_cv2.imread.return_value = None
        elif desired_behaviour =='image':
            mocked_cv2.imread.return_value = ndarray((1024, 768, 3))
        elif desired_behaviour == 'exception':
            mocked_cv2.imread.side_effect = MemoryError('error')
        return mocked_cv2
```
and then use it like:
`mock.patch('verifier.tasks.import_cv2', side_effect=self._prepare_cv2_mock('value')) as mock_import_cv2`

thanks to that, function calls to `imread` are protected and whenever API of the function changes it will be immediately reflected in the tests.